<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>683</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>633</width>
    <height>520</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/concreto.jpg);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>10</y>
      <width>631</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Caracteristicas">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/concreto.jpg);</string>
     </property>
     <attribute name="title">
      <string>Caracteristicas</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>570</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Game</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>471</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Pericias</string>
      </property>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>Conhecimento</string>
       </property>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>96</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QRadioButton" name="Linguas">
           <property name="text">
            <string>Línguas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="linguas">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="Medicina">
           <property name="text">
            <string>Medicina</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="medicina">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="Misticismo">
           <property name="text">
            <string>Misticismo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="misticismo">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>121</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>Manobras</string>
       </property>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="Acrobacia">
           <property name="text">
            <string>Acrobacia</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="acrobacia">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="Furtividade">
           <property name="text">
            <string>Furtividade</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="furtividade">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="Persuasao">
           <property name="text">
            <string>Persuasão</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="persuasao">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>191</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>Subterfugio</string>
       </property>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>171</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QRadioButton" name="DesFechadura">
           <property name="text">
            <string>Destravar Fechadura</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="desfechadura">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="FurObjetos">
           <property name="text">
            <string>Furtar Objetos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="furobjetos">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="SeguirTrilhas">
           <property name="text">
            <string>Seguir Trilhas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="seguirtrilhas">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>599</width>
        <height>302</height>
       </rect>
      </property>
      <property name="title">
       <string>Personagem</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>251</width>
         <height>218</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="Nome">
            <property name="text">
             <string>Nome</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nomeEdit">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="Raca">
            <property name="text">
             <string>Raça</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="racaBox">
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Humano</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pequeninos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Anões</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Elfo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Elfo Florestal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Meio-Elfo</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="Experiencia">
            <property name="text">
             <string>Experiência</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="Estagio">
            <property name="text">
             <string>Estágio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="Proficao">
            <property name="text">
             <string>Profição</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="proficaoBox">
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bardos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Guerreiros</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ladinos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rastreadores</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="proficaoDoisEdit_6"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="Deus">
            <property name="text">
             <string>Deus</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="simDeusButton">
            <property name="text">
             <string>Sim</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>Não</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="face">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>141</width>
         <height>181</height>
        </rect>
       </property>
       <property name="text">
        <string>Imagem da Raça</string>
       </property>
      </widget>
      <widget class="QLabel" name="proficao">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>50</y>
         <width>141</width>
         <height>181</height>
        </rect>
       </property>
       <property name="text">
        <string>Imagem da Profição</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>320</y>
        <width>111</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Atributos</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>227</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="Intelecto">
            <property name="text">
             <string>Intelecto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="intelectoLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="Aura">
            <property name="text">
             <string>Aura</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="auraLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="Carisma">
            <property name="text">
             <string>Carisma</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="carismaLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="Forca">
            <property name="text">
             <string>Força</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="forcaLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="Fisico">
            <property name="text">
             <string>Físico</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="fisicoLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="Agilidade">
            <property name="text">
             <string>Agilidade</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="agilidadeLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="Percepcao">
            <property name="text">
             <string>Percepcao</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="percepcaoLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Autores">
     <attribute name="title">
      <string>Autores</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>621</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>Alex Ladeiras, Alex Resende, Alisson Diorgenes Vasconcelos, Andre Luis Tomaz Dionísio,
 Angelo Gurzoni Junior e Pablo Gallis, Bernard Guimaraes Nogueira, Carlos Martins, Diego Awi,
 Diego Guinâncio, Fábio Carvalho Magalhães, Felipe Deprá Galdino,
 Fernando Nascimento Reis da Costa, Geraldo Lúcio Cabral da Silva,
 Gustavo Lorena Pinto, João Antonio Modernel Bender, Jonas Conrado Cardozo,
 José Jorge Osório da Silva Junior, Marcelo Bouzas Barbosa Teixeira, Marcelo Rodrigues,
 Mauro Draco, Samuel Oliveira de Azevedo, Rywv, Saulo Carvalho Rocha,
 Stefano Guimarães Giusini, Thiago Rodrigues do Nascimento e Wévison Guimarães dos Santos Genuino.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>561</width>
        <height>451</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/unnamed.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
